app/scripts/views/mixins/signin-mixin.js

```diff
@@ -98,6 +103,12 @@ define(function (require, exports, module) {
 
       return this.invokeBrokerMethod(brokerMethod, account)
         .then(this.navigate.bind(this, this.model.get('redirectTo') || 'settings', {}, navigateData));
+    },
+
+    engageForm() {
+      // user has engaged with the signin or sign up form
+      // log 'flow.engage' from both forms because both forms can login.
+      this.logEventOnce('flow.engage');
```

Comment:
only log it once per page reload, which should match new flow.begin events right?